web/thesauruses/javascript/exception_handling.json

```diff
@@ -0,0 +1,386 @@
+    },
+    "are_exception_types_functions": {
+      "name": "Are specific exception types a functions?",
+      "code": "false"
+    },
+    "are_exception_types_objects": {
+      "name": "Are specific exception types an object?",
+      "code": "true"
+    },
+    "are_exception_types_gotos": {
+      "name": "Are specific exception types a Goto statement?",
+      "code": "JavaScript has no goto statement."
+    },
+    "base_exception_type": {
+      "name": "Base exception type name",
+      "code": "Error"
+    },
+    "define_new_exception_type": {
+      "name": "Statement to define a new exception type",
+      "code": "class CustomError extends Error {}"
+    },
+    "try_or_operator": {
+      "name": "Try-Or operator",
+      "not-implemented": "true"
+    },
+    "statement_to_start_trying_code": {
+      "name": "Statement to start a block of code to try",
+      "code": "try {\n  // try statements\n}"
+    },
+    "statement_to_start_catching_exceptions": {
+      "name": "Statement to start catching exceptions",
+      "code": "} catch(error) {  \n  // catch statements\n}"
+    },
+    "statement_to_catch_generic_exception": {
+      "name": "Statement to catch a generic exception",
+      "code": "} catch(error) {  \n  // catch statements\n}"
+    },
+    "statement_to_catch_specific_exception": {
+      "name": "Statement to catch a specific exception",
+      "code": "try {\n  // try statement\n} catch (error) {\n   if ( error instanceof CustomExceptionError ) {\n    // catch statement\n  }\n}"
+    },
+    "statement_to_catch_multiple_exceptions": {
+      "name": "Statement to catch multiple types of exceptions",
+      "code": "try {\n  // try statement\n} catch (error) {\n  if (error instanceof FirstCustomExceptionError) {\n    // catch statement\n  } else if (error instanceof SecondCustomExceptionError) {\n    // catch statement\n  } else {\n    // statement for rest exceptions\n  }\n}\n"
+    },
+    "statement_to_break_out_if_caught_exception": {
+      "name": "Statement to break out of a block if an exception is caught",
+      "not-implemented": "true"
+    },
+    "statement_to_stop_program_if_caught_exception": {
+      "name": "Statement to stop the program if an exception is caught",
+      "code": "} catch(error) {  \n  throw\n}",
+      "comment": "Execution of the current function will stop, and control will be passed to the first catch block in the call stack. If no catch block exists among caller functions, the program will terminate."
+    },
+    "statement_to_start_code_if_no_exceptions_caught": {
+      "name": "Statement to start code if no exceptions are caught",
+      "not-implemented": "true"
+    },
+    "statement_to_start_cleaning_up_after_exception": {
+      "name": "Statement to start code ran after an exception",
+      "code": "try {\n  // try statements\n} catch(error) {\n  // catch statements\n} finally {\n  // finally statements\n}"
```

Comment:
This should just be the statement to clean up, so just `finally`, or possibly `finally { //statements }` kind of a thing.